packages/base-bytes/base-bytes.base+dune/opam

```diff
@@ -15,3 +15,6 @@ url {
   src: "https://github.com/kit-ty-kate/bytes/archive/v0.1.0.tar.gz"
   checksum: "sha256=795b9bf545841714aaf0e517b62834a589937f65ad815ed4589ea56fa614d238"
 }
+post-messages: [
+  "Do not install base-bytes.opam into an opam switch, it will not result in a working bytes library. This package is purely meant to be used with opam-monorepo."
```

Comment:
Can you also add a workaround? For instance, "use `opam install base-bytes.base` explicitly instead"?

Reply:
Done!